meta/halyard

```diff
@@ -7,8 +7,12 @@ set -euo pipefail
 FLAGS=''
 if [ -n "$DEBUG" ] ; then
     FLAGS="$FLAGS --debug"
+    if [ -n "$PROFILE" ]; then
```

Comment:
Trying to find the official docs for ya but, without `--debug`, `--profile` doesn't output anything extra.
Edited `meta/haylard` to pass `--profile` and:

```
❯ halyard
Repo is unclean: /opt/halyard/repo
Warning: Support for ruby version 2.0.0 is deprecated and will be removed in a future release. See https://docs.puppet.com/puppet/latest/system_requirements.html#ruby for a list of supported ruby versions.
   (at /opt/halyard/vendor/ruby/2.0.0/gems/puppet-4.9.2-universal-darwin/lib/puppet.rb:167:in `<module:Puppet>')
Warning: Use of class Puppet::SemVer is deprecated. SemanticPuppet::Version or SemanticPuppet::VersionRange should be used instead
   (file & line not available)
Notice: Compiled catalog for jareds-macbook-pro-2.local in environment production in 0.81 seconds
Notice: Applied catalog in 6.83 seconds
```
But with `export DEBUG=1` we get the DEBUG logs and these bits at the end:

```
Info: AGGREGATE PROFILING RESULTS:
Info: ----------------------------
Info: compiler: 3.053663 s (36 calls)
Info: compiler -> evaluate_resource: 1.366736 s (11 calls)
Info: compiler -> evaluate_resource -> Class[main]: 0.715613 s (1 calls)
Info: compiler -> evaluate_resource -> Class[Homebrew]: 0.46272 s (1 calls)
Info: compiler -> evaluate_resource -> Class[Dotfiles]: 0.143229 s (1 calls)
Info: compiler -> evaluate_resource -> Class[Zsh]: 0.010964 s (1 calls)
Info: compiler -> evaluate_resource -> Sudoers::Allowed_command[cask_installer]: 0.010906 s (1 calls)
Info: compiler -> evaluate_resource -> Class[Halyard]: 0.006556 s (1 calls)
Info: compiler -> evaluate_resource -> Homebrew::Tap[halyard/formulae]: 0.005789 s (1 calls)
Info: compiler -> evaluate_resource -> Sudoers::Allowed_command[halyard_puppet]: 0.004365 s (1 calls)
Info: compiler -> evaluate_resource -> Homebrew::Tap[halyard/casks]: 0.002989 s (1 calls)
Info: compiler -> evaluate_resource -> Homebrew::Tap[homebrew/dupes]: 0.002419 s (1 calls)
Info: compiler -> evaluate_resource -> Class[Sudoers]: 0.001186 s (1 calls)
Info: compiler -> compile: 0.855838 s (1 calls)
Info: compiler -> compile -> production: 0.855838 s (1 calls)
Info: compiler -> compile -> production -> jareds-macbook-pro-2.local: 0.855838 s (1 calls)
Info: compiler -> evaluate_main: 0.715937 s (1 calls)
Info: compiler -> evaluate_generators: 0.029284 s (1 calls)
Info: compiler -> iterate_on_generators: 0.029032000000000002 s (3 calls)
Info: compiler -> evaluate_definitions: 0.027408999999999996 s (3 calls)
Info: compiler -> finish_catalog: 0.020582 s (1 calls)
Info: compiler -> create_settings_scope: 0.005378 s (1 calls)
Info: compiler -> validate_final: 0.001216 s (1 calls)
Info: compiler -> evaluate_collections: 0.001067 s (3 calls)
Info: compiler -> set_node_params: 0.000726 s (1 calls)
Info: compiler -> prune_catalog: 0.000263 s (1 calls)
Info: compiler -> evaluate_capability_mappings: 6.400000000000001e-05 s (2 calls)
Info: compiler -> init_server_facts: 5.7e-05 s (1 calls)
Info: compiler -> evaluate_node_classes: 2.5e-05 s (1 calls)
Info: compiler -> evaluate_site: 2.0e-05 s (1 calls)
Info: compiler -> evaluate_ast_node: 1.3e-05 s (1 calls)
Info: compiler -> validate_pre_finish: 1.1e-05 s (1 calls)
Info: compiler -> evaluate_applications: 5.0e-06 s (1 calls)
Info: functions: 0.7421460000000001 s (24 calls)
Info: functions -> include: 0.66907 s (7 calls)
Info: functions -> create_resources: 0.054965 s (1 calls)
Info: functions -> each: 0.011866 s (2 calls)
Info: functions -> template: 0.0028190000000000003 s (2 calls)
Info: functions -> epp: 0.002528 s (1 calls)
Info: functions -> dotfiles::homedir: 0.000394 s (1 calls)
Info: functions -> split: 0.000383 s (3 calls)
Info: functions -> realize: 6.500000000000001e-05 s (2 calls)
Info: functions -> join: 3.3e-05 s (3 calls)
Info: functions -> concat: 2.3e-05 s (2 calls)
Info: indirector: 0.003299 s (1 calls)
Info: indirector -> filter: 0.003299 s (1 calls)
Info: indirector -> filter -> catalog: 0.003299 s (1 calls)
Info: indirector -> filter -> catalog -> jareds-macbook-pro-2.local: 0.003299 s (1 calls)
Info: ----------------------------
```